src/rospkg/rospack.py

```diff
@@ -387,6 +388,38 @@ def stack_of(self, package):
             else:
                 d = os.path.dirname(d)
 
+    def get_licenses(self, pkg_name, implicit=True):
+        """
+        @summary: Return a list of licenses and the packages in the dependency tree
+            for the given package. Special value 'ERR' is used as the license for the
+            packages that license was not detected for.
+        @param pkg_name: Name of the package the dependency tree begins from.
+        @return Dictionary of license name and a list of packages.
+        @rtype { k, [d] }
+        @raise ResourceNotFound
+        """
+        MSG_LICENSE_NOTFOUND_SYSPKG = "ERR"
```

Comment:
This seems to be a pretty awkward choice for the API. Maybe use the key `None` for package names where no license information is available.